ioservice/cob_foms.c

```diff
@@ -1131,6 +1139,10 @@ static int cd_cob_delete(struct m0_fom            *fom,
 		return M0_RC(rc);
 
 	M0_ASSERT(cob != NULL);
+
+	pver = cob->co_nsrec.cnr_pver;
+	cob_size = cob->co_nsrec.cnr_size;
+
```

Comment:
Check if pver and cob_size are valid or not.

Reply:
Done